sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/ReflectionUtils.java

```diff
@@ -31,6 +31,7 @@
 import com.azure.cosmos.implementation.clienttelemetry.ClientTelemetry;
 import com.azure.cosmos.implementation.cpu.CpuMemoryListener;
 import com.azure.cosmos.implementation.cpu.CpuMemoryMonitor;
+import com.azure.cosmos.implementation.directconnectivity.rntbd.RntbdConnectionStateListener;
```

Comment:
nit: extra import. Not worth blocking on this comment. Feel free to merge without this change and follow it up in next PR.

Reply:
removed.